specs/pvs/corla.pvs

```diff
@@ -486,11 +527,15 @@ END ballot_manifests
 % @relation well_formed_vote inherit vote
```

Comment:
line 476/517  what does this line correspond to in the real world (verified?: pred[ballot_manifest])? In what sense is a ballot manifest verified or not verified?